documentation/modules/ROOT/examples/aspect-with-double-either-characteristic.ttl

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2024 Robert Bosch Manufacturing Solutions GmbH
+
+# See the AUTHORS file(s) distributed with this work for additional information regarding authorship.
+
+# This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
+# If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/
+# SPDX-License-Identifier: MPL-2.0
+
+@prefix : <urn:samm:com.mycompany.myapplication:1.0.0#> .
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:AspectWithEither a samm:Aspect ;
+  samm:properties ( :speedProperty ) ;
+  samm:operations ( ) .
+
+# tag::aspect-with-double-either-characteristic[]
+:speedProperty a samm:Property ;
+  samm:characteristic :Result .
+
+:Result a samm-c:Either ;
+  samm:description "The result is either an error or a SpeedValue"@en ;
+  samm-c:left :Error ;
+  samm-c:right :SpeedValue .
+
+:Error a samm-c:Either ;
+  samm:description "An error can be either a simple text, <br> or a complex object with separate errorCode and errorDescription"@en ;
```

Comment:
`<br>` is neither interpreted nor passed-through by Antora. Just use a literal line break with triple quotes:

```suggestion
  samm:description """An error can be either a simple text,
  or a complex object with separate errorCode and errorDescription"""@en ;
```

This renders correctly in the documentation:
![image](https://github.com/eclipse-esmf/esmf-semantic-aspect-meta-model/assets/1199036/9a51ac87-8d86-4f4d-9eec-f0e0396fe46d)